automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala

```diff
@@ -10,12 +10,15 @@ import org.openqa.selenium.support.ui.{ExpectedConditions, WebDriverWait}
   * checkboxes that represent their own state
   */
 trait Clickable extends LazyLogging { this: Component =>
+
+  val spinner = cssSelector("[data-test-id=spinner]")
+
   /**
     * Click on the element modeled by this Component
     */
   def doClick()(implicit webDriver: WebDriver): Unit = {
     try {
-      await condition (isVisible && isEnabled)
+      await condition ( isVisible && isEnabled && !(find(spinner).exists(_.isDisplayed)) )
```

Comment:
spinner is happening more frequently when server response is slower. this is likely related to network speed on fiab hosted Firecloud app.